modules/oe_translation_poetry/modules/oe_translation_poetry_html_formatter/tests/Kernel/HtmlFormatterTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation_poetry_html_formatter\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+
+/**
+ * Tests the Poetry Html Formatter.
+ */
+class HtmlFormatterTest extends KernelTestBase {
```

Comment:
Rebase the POETRY branch and see if you can extend `TranslationKernelTestBase` and add the extra stuff you may need to yours. Perhaps would make sense to install the more used entity schema's like tmgmt job and node in the base test class as well.